packages/firestore/src/local/local_store.ts

```diff
@@ -183,7 +174,7 @@ export interface LocalStore {
  * This is useful to implement optional features (like bundles) in free
  * functions, such that they are tree-shakeable.
  */
-class LocalStoreImpl implements LocalStore {
+export class LocalStoreImpl implements LocalStore {
```

Comment:
Same comments regarding the changes here as in SyncEngine. In general, since we don't seem to need the source re-arrangement anymore (with the prebuilt), we should move the bundle code back to where it was before.

Reply:
Done.